### ISSUE TITLE
🐛 Bug Report: `GraphQL` Service Missing

### ISSUE THREAD

**Issue by ItzNotABug** [appwrite/appwrite#6501]
### 👟 Reproduction steps

GraphQL is missing from both Cloud & Self-Hosted instance but is received in the json data fetched for Settings screen.
I can see the following for Services in the json response - 
```json
{
  "serviceStatusForAccount":true,
  "serviceStatusForAvatars":true,
  "serviceStatusForDatabases":true,
  "serviceStatusForLocale":true,
  "serviceStatusForHealth":true,
  "serviceStatusForStorage":true,
  "serviceStatusForTeams":true,
  "serviceStatusForUsers":true,
  "serviceStatusForVcs":true,
  "serviceStatusForFunctions":true,
  "serviceStatusForProxy":true,
  "serviceStatusForGraphql":true,
  "serviceStatusForMigrations":true
}
```

Also, the `VCS` & `Proxy` are also missing if they are supposed to be added in the Console.

---
The source - https://github.githistory.xyz/appwrite/console/blob/ffb7476b28e55b8fbffb57626bfa4f8cdbdaf821/src/lib/stores/project-services.ts#L4

### 👍 Expected behavior

Show all supported Services.

### 👎 Actual Behavior

Supported services are missing.

### 🎲 Appwrite version

Appwrite Cloud

### 💻 Operating system

Linux

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
@ItzNotABug, thanks for raising this issue! 🙏🏼 Let me check with the team on this.

**Comment by stnguyen90** [comment]
We want to have a separate section in the UI for REST API, GraphQL and Realtime so our design team will have to come up with something for this first.

**Comment by ItzNotABug** [comment]
@stnguyen90 Until then, GraphQL will anyway stay enabled and active, right? Not that it is a major concern but just curious.

**Comment by eldadfux** [comment]
> [@stnguyen90](https://github.com/stnguyen90) Until then, GraphQL will anyway stay enabled and active, right? Not that it is a major concern but just curious.

Yes it will, if you want to get sophisticated you can still use the API to block it.
